src/encoded/schemas/experiment_type.json

```diff
@@ -0,0 +1,151 @@
+{
+    "title": "Experiment Type",
+    "description": "Schema for experiment type",
+    "id": "/profiles/experiment_type.json",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "type": "object",
+    "required": ["title"],
+    "identifyingProperties": ["experiment_name", "uuid", "aliases"],
```

Comment:
If we don't want to support aliases for these then should remove aliases from identifyProperties. 
Alternatively it doesn't hurt to have aliases so could add the mixin.